cisco_aci/datadog_checks/cisco_aci/ndm.py

```diff
@@ -48,9 +48,11 @@ def create_interface_metadata(phys_if, address, namespace):
     eth = PhysIf(**phys_if.get('l1PhysIf', {}))
     interface = InterfaceMetadata(
         device_id='{}:{}'.format(namespace, address),
-        id_tags=['interface:{}'.format(eth.attributes.name)],
+        raw_id=eth.attributes.id,
+        id_tags=['interface:{}'.format(eth.attributes.id)],
```

Comment:
Is the ACI part for NDM already in use by customers? If we are using `name` currently and switch the `id_tags` to `id` I think this would break metric linkage to older metrics when a customer upgrades.

Reply:
so the funny thing is, and this is sort of buried but if an empty string is provided for name (which in this case it is, i guess that's a default?) i made it so that it would utilize the `id`

so given that the `raw_id` would utilize the `id`, i decided we should just prioritize that ... but i guess if name is utilized we'd want the id? :thonk:

hmm i guess it makes sense to keep the name, given if it isn't provided it would just be the `id` format anyways

Reply:
Just to make sure I understand how this was before this change, am I right in saying the following given two example interfaces?

Interface 1:
```yaml
eth.attributes.name: "iface1_eth_name"
eth.attributes.id: "eth1/1"
```

`id_tags` would be `"interface:iface1_eth_name"`

Interface 2:
```yaml
eth.attributes.name: ""
eth.attributes.id: "eth2/1"
```

`id_tags` would be `"interface:eth2/1"`

For interface 2 in the example, I agree it's not an issue but it would be for interface 1. Are you saying that we don't see an example like interface 1 today?

Reply:
yes! for example like interface 1, we don't see it on any of the sandboxes (i guess the default in sandboxes is always empty names?)

i'm not sure if that's the default always since i've never interacted with a real ACI environment / to get the response from the interfaces, but i'm deferring to the original as you suggested since, that would take precendence, and if there _were_ no name, then it'd show the id anyways, so nothing really lost!